lib/index.js

```diff
@@ -19,25 +17,22 @@ const logo = new LabIcon({
   id: 'logo-id'
 });
 
-class ExampleWidget extends Widget {
-  constructor() {
-    super();
-    this.id = 'simple-widget-example';
-    // this.title.label = 'Sirepo Support';
-    this.title.icon = logo;
-  }
+class SlackWidget extends Widget {
+    constructor() {
+        super();
+        this.id = 'slack-widget';
+        this.title.icon = logo;
+        this.node.classList.add('slack-logo');
+    }
 }
 
-
-async function getUrlWithUID(Url) {
-  fetch(Url, {
+function getSlackUrlWithUid() {
+  return fetch('/auth-state', {
```

Comment:
I'd add 
```
// POSIT: sirepo.simulation_db.SCHEMA_COMMON.route.authState
```

Usually when I search for routes I search for their key name (authState) not the uri (auth-state). Adding this we are at least a little closer to catching this use of authState which is outside of the Sirepo app and likely to be missed if we are ever looking for all uses.

Reply:
46e3f029c06bba1657a5a6253d3a223fcf5da9ca